### PR TITLE
Fix installation with RSA/PSS

### PR DESCRIPTION
**Pull request by edewata**
The `pki ca-cert-import` and `pki nss-cert-import` CLIs have been modified to parse the cert data after initializing JSS to ensure that it works with RSA/PSS.

Doc: https://github.com/edewata/pki/blob/rsa-pss/docs/installation/ca/Installing-CA-with-RSA-PSS.adoc

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2083575

### CODE REVIEW

**Review by fmarco76 — APPROVED** [review]
LGTM

File: base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java [hunk 1]
```diff
@@ -84,11 +84,12 @@ public void execute(CommandLine cmd) throws Exception {
             throw new Exception("Unsupported format: " + format);
         }
```
Comment (ckelleyRH) [inline base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java:85]: Slightly less of a problem here, but for consistency would be nice to do something similar.

**Review by ckelleyRH — APPROVED** [review]
Change is fine, just one tidy up suggested.

Outside scope for this PR - these execute methods are really long and confusing. There are many different things happening and they are interleaved, so it is hard to follow. Using this as an example, we build a cert from a byte array on (old) line 115, but it doesn't actually get used until line 171 - we do 50 lines of other things. Refactoring required here I think.

File: base/ca/src/main/java/org/dogtagpki/server/ca/cli/CACertImportCLI.java [hunk 2]
```diff
@@ -112,8 +112,6 @@ public void execute(CommandLine cmd) throws Exception {
             throw new Exception("Unsupported format: " + certFormat);
         }
```
[the remark this answers is not in the document]
Comment (edewata) [reply base/ca/src/main/java/org/dogtagpki/server/ca/cli/CACertImportCLI.java:131]: Yes, I've moved the `byte[]`-related code down, but git thought that I moved JSS initialization up, so the patch generated by git doesn't really reflect the changes I'm making, but that should be fine.

Since we need to cherry-pick these changes to `v11.2` branch I'd rather keep them simple, but we can do more cleanups in the `master` branch later.

File: base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java (hunk 1 above)
Comment (edewata) [reply base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java:90]: Done.

**Comment by edewata** [comment]
@fmarco76 @ckelleyRH Thanks!
